pycentral/device_inventory.py

```diff
@@ -69,3 +69,59 @@ def get_inventory(self, conn, sku_type="all", limit=0, offset=0):
 
         resp = conn.command(apiMethod="GET", apiPath=path, apiParams=params)
         return resp
+
+    def archive_devices(self, conn, device_serials=[]):
+        """Archive a list of devices using serial numbers 
+
+        :param conn: Instance of class:`pycentral.ArubaCentralBase` to make an API call.
+        :type conn: class:`pycentral.ArubaCentralBase`
+        :param device_serials: List of serial number of Aruba devices that should be archived
+        :type device_serials: list
+        :return: Response as provided by 'command' function in class:`pycentral.ArubaCentralBase`
+        :rtype: dict
+        """
+        path = urls.DEVICES["ARCHIVE_DEVICES"]
+        if isinstance(device_serials, str):
+            device_serials = [device_serials]
+        if len(device_serials) > 0:
+            apiData = {
+                "serials": device_serials
+            }
+            resp = conn.command(apiMethod="POST", apiPath=path, apiData=apiData)
+            return resp
+    
+    def unarchive_devices(self, conn, device_serials=[]):
+        """Unarchive a list of devices using serial numbers 
+
+        :param conn: Instance of class:`pycentral.ArubaCentralBase` to make an API call.
+        :type conn: class:`pycentral.ArubaCentralBase`
+        :param device_serials: List of serial number of Aruba devices that should be unarchived
+        :type device_serials: list
+        :return: HTTP Response as provided by 'command' function in class:`pycentral.ArubaCentralBase`
+        :rtype: dict
+        """
+        path = urls.DEVICES["UNARCHIVE_DEVICES"]
+        if isinstance(device_serials, str):
+            device_serials = [device_serials]
+        if len(device_serials) > 0:
+            apiData = {
+                "serials": device_serials
+            }
+            resp = conn.command(apiMethod="POST", apiPath=path, apiData=apiData)
+            return resp
+    
+    def add_devices(self, conn, device_details):
+        """Add device(s) using Mac & Serial Numbers
+
+        :param conn: Instance of class:`pycentral.ArubaCentralBase` to make an API call.
+        :type conn: class:`pycentral.ArubaCentralBase`
+        :param device_details: List of dictionaries with Aruba devices that should be added to the Aruba Central account
```

Comment:
need to specify the keys of the dictionary in documentation for clarity to users.
  {
    "mac": "test",
    "serial": "test",
    "partNumber": "test"
  }